src/components/pages/settings/LayersFormContainer.js

```diff
@@ -0,0 +1,24 @@
+import { connect } from 'react-redux';
+import LayersForm from './LayersForm';
+import { getGFWLayers, createLayer } from '../../../modules/layers';
+
+
+const mapStateToProps = ({ layers, teams }) => {
+    return { 
```

Comment:
This again is point of preference, but as there are not a lot of props, and similar things are getting passed to LayerForm and LayerSwitcher, we could have a single Container to the LayersForm (their parent), and then pass the need props to each. This leaves the Switcher and Form as UI components and you can control the state of each more clearly. Lets discuss.